s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -121,14 +123,33 @@ export class MapComponent implements OnInit, OnDestroy {
       source: this.markerSource,
       style: markerStyle,
     }));
+
+    this.map.on('moveend', this.onMoveEnd);
   }
 
   ngOnDestroy(): void {
     this.activeScene$.complete();
     this.selectedLocationSub.unsubscribe();
     this.selectedLocationSub = null;
+    this.map.un('moveend', this.onMoveEnd);
```

Comment:
make `onMoveEnd` a method and call it here like this:

`this.map.un('moveend', e => this.onMoveEnd(e));`